gabi_test.go

```diff
@@ -215,7 +215,7 @@ func TestTestKeys(t *testing.T) {
 
 func TestCLSignature(t *testing.T) {
 	m := []*big.Int{big.NewInt(1), big.NewInt(2), big.NewInt(3)}
-	sig, err := SignMessageBlock(testPrivK, testPubK, m)
+	sig, err := SignMessageBlock(common.NewRandomStorage(), testPrivK, testPubK, m)
```

Comment:
The tests currently fail, fixed in https://github.com/minvws/gabi/pull/3 (although that also changes the interface of `NewRandomPool`